webapp/css/netpyne.less

```diff
@@ -569,3 +568,94 @@ div.flexlayout__layout {
   overflow-y: auto;
   flex: 1;
 }
+//experiment table start
+.experimentTableContainer {
+  &::-webkit-scrollbar {
+    height: 8px;
+  }
+  .experimentTable {
+    font-family: 'Roboto', sans-serif;
+    .experimentHead {
+      font-size: 1rem;
+    }
+    .experimentHeadIcon {
+      font-size: 1rem;
+    }
+    .experimentDate {
+      font-style: italic;
+      color:#A8A5A5;
```

Comment:
Hardcoded color used multiple times -> add to `variables.less`

We already have many shades of gray in Netpyne, would rather reuse an existing one. (@ddelpiano sth to discuss with Paul, want to keep the color theme consistent)